src/components/table/SnapshotAccessRequestTable.tsx

```diff
@@ -58,7 +59,13 @@ function SnapshotAccessRequestTable({
     {
       label: 'Request Name',
       name: 'snapshotName',
-      render: (row: SnapshotAccessRequestResponse) => row.snapshotName || '',
+      render: (row: SnapshotAccessRequestResponse) => (
+        <ModalTrigger
+          modalText={row.snapshotName}
```

Comment:
I don't think this is the "modal text". It's the prompt which is a link to the modal. So maybe `linkText` or `triggerPrompt`, if the name `trigger` is kept.